tests/python/rest_api/utils.py

```diff
@@ -68,9 +412,14 @@ def _get_field(cls, d: Dict[str, Any], path: Union[str, FieldPath]) -> Optional[
         assert path
         for key in path:
             if isinstance(d, dict):
+                assert isinstance(key, str)
                 d = d.get(key)
             else:
-                d = None
+                if callable(key):
```

Comment:
I think the definition of `FieldPath` needs to be amended to account for this.

Reply:
It has already been updated https://github.com/cvat-ai/cvat/blob/1f37236e09844f97716ba147595fcd3c6d30055f/tests/python/rest_api/utils.py#L392

Reply:
My bad, must've missed it.